bsyncpy/bsync.py

```diff
@@ -16325,7 +16327,17 @@ class HVACSystemType(BSElement):
 
 # BuildingType
 class BuildingType(BSElement):
```

Comment:
@JoelBender -- What version of BSync.py is this attached too, I'm guessing `v3.0.0-pr1`, is that right?

Reply:
Yes, that is correct.  A while ago Cory mentioned that it would be beneficial to have the version number in the module someplace along with the XML document that it builds, I'm not quite sure how I would do that.

Reply:
gotcha, thanks! Yes, it would be great to figure out how to handle specific versions but for now we can handle it based on the release versions. I think I'm going to point this one to develop-v3 in order to delineate the BSync 2.x and 3.x work.